src/templates/next/components/Button/Button.tsx

```diff
@@ -1,45 +1,120 @@
 import { ButtonProps } from "~/common"
-import { NextButtonColorVariant } from "~/common/Button"
+import { NextButtonColorScheme } from "~/common/Button"
 import { SUPPORTED_ICONS_MAP } from "~/common/Icons"
 
-const colorVariantToClassMap: Record<NextButtonColorVariant, string> = {
-  black: "bg-[#333333] text-white",
-  white: "bg-white text-[#333333]",
-}
-
-const Button = ({ label, colorVariant, href, rightIcon }: ButtonProps) => {
-  const Label = () => (
-    <span className="text-center text-lg font-semibold leading-tight">
-      {label}
-    </span>
-  )
+const Label = ({ label }: Pick<ButtonProps, "label">) => (
+  <span className="text-center text-lg font-semibold leading-tight">
+    {label}
+  </span>
+)
 
-  const RightIcon = () => {
-    if (!rightIcon) {
-      return null
-    }
-    const Icon = SUPPORTED_ICONS_MAP[rightIcon]
-    return (
-      <div>
-        <Icon className="size-6" />
-      </div>
-    )
+const RightIcon = ({ rightIcon }: Pick<ButtonProps, "rightIcon">) => {
+  if (!rightIcon) {
+    return null
   }
+  const Icon = SUPPORTED_ICONS_MAP[rightIcon]
+  return (
+    <div>
```

Comment:
this isn't an issue but just curious why we need a `div` here

Reply:
originally added because the icon was shrinking on smaller screens, but fixed

[1f67649](https://github.com/isomerpages/isomer-components/pull/22/commits/1f6764961d958f58a61e6e1d00c1f6f67d572bf8)